src/components/organisms/UserProfileModal/UserProfileModal.tsx

```diff
@@ -67,7 +75,7 @@ export const UserProfileModal: React.FC<UserProfileModalProps> = ({
 
             return (
               <React.Fragment key={question.text}>
-                <p className="light question">{question.text}</p>
+                <p className="light no-margin">{question.text}</p>
                 <h6>{questionAnswer}</h6>
               </React.Fragment>
             );
```

Comment:
[OOS] But the usage of `h6` here is SUPER  janky. Header tags shouldn't be used for styling like this. This breaks all sorts of accessibility rules.

Reply:
Actually.. bootstrap has classes that mirror the header styling. Can you change this to a div or something and use the h6 class at the very least? Should be a quick fix for it i'm guessing.

Reply:
Why are they junky though? Note that this is for `env/github` only

Reply:
Not sure which bootstrap you are referring to, but it looks like here the h1 tags are also used.
https://mdbootstrap.com/docs/standard/navigation/headers/

Reply:
'Bootstrap' refers to https://getbootstrap.com/

- https://getbootstrap.com/docs/4.0/content/typography/#headings
  - > `.h1` through `.h6` classes are also available, for when you want to match the font styling of a heading but cannot use the associated HTML element.

Reply:
Done in https://github.com/sparkletown/sparkle/pull/1667/commits/41dc9b23e005d6755989c6684484c7568675d1d9

Out of curiousity, hoes does it effect the `all sorts of accessibility rules`?

Reply:
I've explained this way back in the past. But basically headings have a very specific and defined semantic meaning in HTML structure, that is used by screen readers/etc.

This was from a quick google, there may be other/better resources:

- https://www.w3.org/WAI/tutorials/page-structure/headings/